src/app/classes/response-rules-interpreter.ts

```diff
@@ -67,7 +75,7 @@ export class ResponseRulesInterpreter {
       return value.includes(rule.value);
     }
 
-    return value === rule.value;
+    return value.toString() === rule.value.toString();
```

Comment:
apparently, when an env is exported, numbers get converted to string, i.e. ("foo": "1"). when another dev imported the json, the rule no longer worked. i'm using toString() - not a fan of "==" - to ensure that numbers work as rule as well.